src/main/java/seedu/address/logic/commands/update/UpdateCancelCommand.java

```diff
@@ -0,0 +1,45 @@
+package seedu.address.logic.commands.update;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.model.Model.PREDICATE_SHOW_ALL_PROPERTIES;
+
+import seedu.address.commons.core.Messages;
+import seedu.address.commons.core.index.Index;
+import seedu.address.logic.commands.CommandResult;
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.property.Property;
+import seedu.address.model.property.status.Status;
+
+public class UpdateCancelCommand extends UpdateCommand {
+
+    public static final String MESSAGE_NULL_STATUS = "Property does not have a status";
```

Comment:
Will you like to move this to the `UpdateCommand` parent class instead so that it is not duplicated in `UpdateProceedCommand` class?

Reply:
moved it! thanks!